### ISSUE TITLE
styleUrls in Beta.3

### ISSUE THREAD

**Issue by Azayzel** [angular-fullstack/generator-angular-fullstack#2451]
So I was able to serve this project only after removing
```
styles: [require('./style.css')]
```
and adding
```
styleUrls: ['./style.css']
```

and fixing the statically typed .scss imports (which i know we are aware of),
from all components that implement it. Before it wouldn't load at all. Is there a reason it was done this way? i'm planning on doing this to the .html files also. 

Also, what is the repo i can push changes to to help with the Angular(2) release?

**Comment by Azayzel** [comment]
closing: found ng-hybrid branch.

**Comment by Awk34** [comment]
Might be related to the css-loader issue that was fixed in 4.1.2 recently.

The server won't serve up any of your CSS/SCSS/etc or HTML/Pug files. We `require` the files and use any transpilation loaders combined with the raw loader to output the resulting text. Basically your CSS/HTML files are inlined as strings for `styles`/`template`. This also helps to eliminate any extra HTTP requests.

The *canary* branch runs Angular 2.
